documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -7,7 +7,7 @@
 
 Strimzi can be upgraded to version {ProductVersion} to take advantage of new features and enhancements, performance improvements, and security options.
 
-As part of the upgrade, you upgrade Kafka to the latest supported version.  
+As part of the upgrade, you upgrade Kafka to the latest supported version.
```

Comment:
This sentence sounds really weird to me. What about something like

`After performing upgrade instructions, you will end up with latest supported version of Kafka.`
? Feel free to scratch this idea, if it's not applicable.

Reply:
It is not AMQ Streams and there are no supported versions here ;-)

Reply:
Perhaps this could be clearer. The upgrade procedure for AMQ Streams involves upgrading Kafka to the latest version. 

That's what we're trying to say.

Reply:
Yes, thank you @laidan6000 
Maybe ` As part of the upgrade, you will upgrade Kafka to the latest supported version as well.`? So not just Cluster operator.